src/components/Quiz/useQuizStore.ts

```
@@ -172,67 +172,72 @@ export const useQuizStore = create<QuizState>()(
 
         // Quiz Actions
         selectAnswer: (questionId, answerId) => {
-          const { selectedAnswers, skippedQuestions, addUserAnswer } = get();
-          // Remove from skipped questions if it was skipped
-          const newSkippedQuestions = skippedQuestions.filter(
-            (id) => id !== questionId
+          const { bySimulado, byAtividade, byAula, skippedQuestions } = get();
+          const quiz = bySimulado || byAtividade || byAula;
+          
```

ESLint

[error] Delete `··········`

(prettier/prettier)